src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -269,7 +269,7 @@
 <script lang="ts">
 import { Component, Emit, Mixins, Prop, Watch } from 'vue-property-decorator'
 import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
-import { OrgPerson } from './'
+import OrgPerson from './OrgPerson.vue'
```

Comment:
The old code imports from the index file in the current folder.
The new code is more explicit.

I'm OK either way.

Reply:
The old code will case the unit test to fail with `TypeError: Cannot read properties of undefined (reading 'default')`. That's why I changed it, otherwise I wouldn't.